k8s/operators/src/pool/diskpool/client.rs

```diff
@@ -1,32 +1,61 @@
-use super::crd::{DiskPool, DiskPoolSpec};
+use super::{
+    crd::DiskPool,
+    new_crd::{DiskPool as NewDiskPool, DiskPoolSpec as NewDiskPoolSpec},
```

Comment:
Instead Of NewDiskPool alias should we rather a versioned alias for better code readability? `V1beta1DiskPool`

Reply:
I think we should